docker/tds/docker-compose.yml

```diff
@@ -1,5 +1,20 @@
-version:  '2'
+version:  '2.1'
```

Comment:
It looks like there is a version 3?  Should we go ahead and update to the latest?